metal-tools-jshexviewer/src/main/java/nl/gertjanal/metaltools/jshexviewer/JsHexViewer.java

```diff
@@ -154,7 +155,9 @@ private static void step(final ParseItem item, final Map<Long, LinkedList<Defini
 		if (!item.isGraph()) {
 			if (item.getDefinition() instanceof Def) {
 				final ParseValue value = item.asValue();
-				getList(map, new Definition(value));
+				if(value.slice.source instanceof ByteStreamSource) {
+					getList(map, new Definition(value));
+				}
```

Comment:
So the only thing needed here for full `Tie`-support is implementing the `else`-clause :-)